src/app/wallet/overview/widgets/coldstake/coldstake.component.ts

```diff
@@ -64,6 +65,9 @@ export class ColdstakeComponent implements OnInit {
     if (this.coldstakeProgress < 100) {
       setTimeout(this.rpc_progressLoop.bind(this), 1000);
     }
+    if (this.coldstakeProgress === 100) {
+      this.activation = 'Activated';
```

Comment:
we still need to poll here in case there's a new transaction after full activation